index.html

```diff
@@ -0,0 +1,190 @@
+                    <div class=""><p class="pclass">Learn Javascript through various internation Speaker around the world and enjoy the best learning experience.</p></div>
+                </div>
+                <div class="main-program-box col-lg-2 col-md-2 col-sm-12 d-flex mb-3 align-items-center ">
+                    <div><img class="lang-img" src="images/logo4.svg" alt="htmllogo"></div>
+                    <div class=""><h2 class='h2class'>South Africa</h2></div>
+                    <div class=""><p class="pclass">Learn React through various internation Speaker around the world and enjoy the best learning experience.</p></div>
+                </div>
+                <div class="main-program-box col-lg-2 col-md-2 col-sm-12 d-flex mb-3 align-items-center ">
+                    <div><img class="lang-img" src="images/logo5.svg" alt="htmllogo"></div>
+                    <div class=""><h2 class='h2class'>Zambia</h2></div>
+                    <div class=""><p class="pclass">Learn Rails through various internation Speaker around the world and enjoy the best learning experience.</p></div>
+                </div>
+                <div class=" col-lg-2 col-md-2 col-sm-12 d-flex mb-3 align-items-center ">
+                    <button class="main-program-btn" type="submit">Enroll Now in Course Batch 2023</button>
+                </div>
+                <div class=" col-lg-2 col-md-2 col-sm-12 d-flex mb-3 align-items-center seeprogramclass">
+                    <h2>See the whole program</h2>
+                </div>
+            </div>
+        </div>
+
+    </div>
+     <!-- Feature section start from here -->
+
+     <div id="feature-section" class="container-fluid feauture-program">
+  
+     </div>
+   <!-- Partner section start from here -->
+    <div id="partner-section" class="container-fluid partner-section">
+        <div class="container partner-program-container ">
+            <div class="row">
+                <div class="col-12 d-flex justify-content-center partner-main-text">Partners</div>
+            </div>
+            <div class="row d-flex justify-content-center">
+                <div class="col-1 partner-main-text-line">line</div>
+            </div>
+        </div>
+
+        <div class="container partner-img-container">
+            <img class="img1" src="images/harvardlogo.png" alt="mozilla">
+            <img class="img2" src="images/europeunilogo.png" alt="airbnb">
+            <img class="img3" src="images/Applelogo.png" alt="daumkakao">
+            <img class="img4" src="images/mercedes1.png" alt="daumkakao">
+
+        </div>
+    </div>
+    <footer id="footer" class="continer-fluid footer-section">
+        <div class="contianer footer-container">
+          <img  class="footer-img"
+            src="images/logo.svg"
+            alt="African cultural dance logo"
+          />
+          <div class="footer-para">2023 African Cultural Dance Foundation. Some Rights Reserved</p>
+        </div>
+      </footer>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please `style` your project using the [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) provided by `Microverse`. Your project must be the same as [Behance's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), you must make your project `color`, `shape`, `layout`, and `fonts` as shown on [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Please correct them🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/235015881-3af3b403-f96e-4e8e-b18a-a6dd4ced7aff.png)